packages/manager/apps/key-management-service/src/data/hooks/useCreateOkmsServiceKey.ts

```diff
@@ -25,6 +29,9 @@ export const useCreateOkmsServiceKey = ({
       return createOkmsServiceKeyResource({ okmsId, data });
     },
     onSuccess: async () => {
+      queryClient.invalidateQueries({
```

Comment:
```suggestion
      await queryClient.invalidateQueries({
```
to wait for the list to be updated before navigate to it.